### DISCUSSION TITLE
Internal filtration for ABS

### DISCUSSION THREAD

**Discussion by Avpman2** [xblax/flashforge_ad5m_klipper_mod#190] [Q&A]
I print a lot of ABS. When I select ABS as a filament, the FW mod doesn't (I'm not sure if it's supposed to) enable the internal filtration fan, and I have to do it manually from the panel. Is it supposed to enable the filter fan, or is there a way to add it to a config somewhere to auto-enable it when I print ABS?

**Comment by consp** [comment]
Most slicers support custom gcode for each material (you'd have to look that up for your specific slicer settings).

Mine calls  AIR_CIRCULATION_INTERNAL macro on ABS and PETG start, my pla one for instance enables the chamber internal fan since it likes to be cooled.
